presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -986,9 +986,9 @@ VeloxQueryPlanConverterBase::toVeloxQueryPlan(
       VELOX_UNSUPPORTED("Unsupported aggregation step");
   }
 
-  bool streamable = !node->preGroupedVariables.empty() &&
-      node->groupingSets.groupingSetCount == 1 &&
-      node->groupingSets.globalGroupingSets.empty();
+  /// TODO karteekmurthys@ Re-enable after
```

Comment:
Nit : There isn't a need to surface this in external documentation.

Just using // is fine.

Reply:
Done.